pages/enterprise/cloud_forwarder.rst

```diff
@@ -0,0 +1,420 @@
+
+As you create your Cloud Forwarder, you must create an Input Profile from the wizard, explained in Add Inputs. 
+Provide a descriptive name for the Input Profile, a short description of its intention, and then create all the input 
+necessary for the Forwarder.
+
+*************
+Configuration
+*************
+
+File
+====
+
+On the client-side of your Forwarder, modify the configuration file to address your Graylog host and authentication. 
+The default values for ``forwarder.conf`` are:
+
+* ``forwarder_server_hostname``: (required) The Graylog Cloud Forwarder ingest hostname (eg. ``ingest-<your-account>.graylog.cloud``). It’s found on the Forwarder Setup Wizard in Graylog Cloud.
+* ``forwarder_grpc_api_token``: (required) The API Token for authenticating the Forwarder.
+
+Here’s an example of the configuration file, with the following values:
+
+.. code-block:: bash
+
+    $ forwarder_server_hostname = ingest-<your-account-url>.graylog.cloud
+    $ forwarder_grpc_api_token = <your-api-token>
+
+Wizard
+======
+
+When you login to the Cloud instance, you’ll walk through a guided process to configure your Forwarder.
+
+.. image:: /images/forwarder_intro_wizard.png
+
+Complete the wizard by following the sections below:
+
+#. Go to the Graylog user interface to select *Forwarders*, found in the *System* menu.
+#. Click the *New Forwarder* button.
+#. Press the *Start configuration* button to navigate to the *Select Forwarder* accordion.
+
+Install Forwarder
+-----------------
+
+#. Download the Forwarder.
+#. Press *Continue* to navigate to the next step in the accordion.
+
+Create API Token
+----------------
+
+We recommend using a single API token for each Forwarder, ensuring that you can revoke tokens for 
+individual Forwarders no longer used.
+
+.. image:: /images/forwarder_api_token.png
+
+#. Enter your *Token Name* in the available field.
+#. Create the new name by clicking the *Create Token* button.
+
+Configure Forwarder (Accordion/Wizard Menu)
+-------------------------------------------
+
+.. image:: /images/forwarder_config.png
+
+#. Open your text editor of choice to access your ``forwarder.conf`` file
+#. Click *Copy as configuration snippet* to copy the configuration details displayed to your ``forwarder.conf`` file, i.e. ``forwarder_server_hostname`` and ``forwarder_grpc_api_token`` with corresponding values.
+#. Paste all values in the configuration file, and save the new changes.
+
+Start the Forwarder
+-------------------
+
+Before you begin the last step in the accordion:
+
+#. Open your terminal
+#. Locate the startup script found in your application directory. 
+#. Run the startup script with the command ``./bin/graylog-forwarder run --configfile forwarder.conf``. 
+#. Return to the wizard after the Forwarder successfully runs.
+
+Select Forwarder
+----------------
+
+Once the Forwarder makes a connection to Graylog Cloud, see it listed among your routers. 
+
+.. image:: /images/forwarder_select.png
```

Comment:
There are quire a lot of Forwarders listed here, and several have the "instance" hostname. What do you think about removing some of those Forwarders and taking the screen shot again with only one Forwarder so that it shows a bit more clearly that the new Forwarder shows up there?